admin/js/boldgrid-backup-admin-backup-now.js

```diff
@@ -155,10 +155,14 @@ BOLDGRID.BACKUP.BackupNow = function( $ ) {
 		 * not refresh until we know a backup is in progress so that we know the in progress bar will
 		 * show when the page refreshes.
 		 */
-
-		setTimeout( function() {
-			location.reload();
-		}, 6000 );
+		console.log( pagenow );
+		if ( true === pagenow.includes( 'boldgrid-backup' ) ) {
```

Comment:
I have code that checks if we're on a Total Upkeep page: https://github.com/BoldGrid/boldgrid-backup/blob/master/admin/class-boldgrid-backup-admin.php#L243-L267

If you need it in JS, you can use the JS translation feature to make it available. IE you can add translations here and include a value for is total upkeep page - https://github.com/BoldGrid/boldgrid-backup/blob/d6fc50fd53de130ab927d3c9ddabb17e50ffa9bc/admin/class-boldgrid-backup-admin-auto-rollback.php#L210-L229

Reply:
How is that any better than using the information already passed in the pagenow variable? Either way we we would need logic in the javascript to run this IF it's a total-upkeep page. Since the data is already being passed, i figured it would be a bit redundant to add a JS translation just for this.

Reply:
Hmm... I was under the impression that not every page had "boldgrid-backup" in it. But, I did another check, and it looks like they do. Ignore this one ;)